apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/Tutorial/index.tsx

```diff
@@ -710,15 +715,31 @@ export default function Tutorial({ useCase }: Props) {
         search: search ? `?${search}` : undefined,
         state: { deviceId: connectedDevice?.deviceId },
       };
-      dispatch(saveSettings({ hasCompletedOnboarding: true }));
+
+      const userId = getUserId();
+      const recoverOnboardingStatus = await getStoreValue(
+        `USER_RESTORE_ONBOARDING_STATUS-${userId}`,
+        recoverStoreId,
+      );
```

Comment:
I think you don't need `await` with `getStoreValue` on LLD, you can also remove the `async` on the function
```suggestion
      const recoverOnboardingStatus = getStoreValue(
        `USER_RESTORE_ONBOARDING_STATUS-${userId}`,
        recoverStoreId,
      );
```